test-data/everest/math_func/jobs/fail_simulation.py

```diff
@@ -19,7 +19,7 @@ def main(argv):
     arg_parser.add_argument("--fail", type=str)
     options, _ = arg_parser.parse_known_args(args=argv)
 
-    if options.fail in os.getcwd():
+    if options.fail is not None and options.fail in os.getcwd():
```

Comment:
I am unsure this change is needed.

Reply:
Right, but otherwise this will fail all the time (ie. each realization), at least that's what I got when testing it